engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java

```diff
@@ -348,13 +348,13 @@ public int size()
 
   public Set<String> getAllDTInstantiableOperators()
   {
-    TypeGraphVertex tgv = typeGraph.get(Operator.class.getName());
+    TypeGraphVertex tgv = typeGraph.get(DAG.GenericOperator.class.getName());
     if (tgv == null) {
       return null;
     }
     Set<String> result = new TreeSet<>();
     for (TypeGraphVertex node : tgv.allInstantiableDescendants) {
-      if ((isAncestor(InputOperator.class.getName(), node.typeName) || !getAllInputPorts(node).isEmpty())) {
+      if ((isAncestor(DAG.GenericOperator.class.getName(), node.typeName) || !getAllInputPorts(node).isEmpty())) {
```

Comment:
Can you explain why this change? It looks like it's changing the intention of the original code.